mscore/timeline.h

```diff
@@ -73,16 +72,16 @@ class TRowLabels : public QGraphicsView {
             };
 
    private:
-      TDockWidget* scrollArea;
+      TDockWidget* _scrollArea;
```

Comment:
You can also initialize

Reply:
It is initialized in the constructor, do you mean adding `{ nullptr }` here? I don't see the need actually.

Reply:
If you add `{ nullptr }` here, then it will be more consistent, in most places we strive for such initialization, because it is understandable and reliable, it is immediately clear that there is initialization, without having to look at the constructor

Reply:
> without having to look at the constructor

But the proper initialization is most probably not `nullptr`, this is what you'll get from the constructor.
Anyway, I'll also add initialization here.